python/chemiscope/sphinx/nodes.py

```diff
@@ -0,0 +1,66 @@
+import os
+import re
+import uuid
+
+from docutils import nodes
+
+
+class chemiscope(nodes.Element):
+    pass
+
+
+def visit_chemiscope_latex(self, node):
+    pass
```

Comment:
should we at least leave some indication in the LaTeX document that we ignored a chemiscope file?

Reply:
Nice to have